docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -94,19 +114,37 @@ $number-of-card-columns-xs-max: 1;
 
 .components {
```

Comment:
Something isn't working correctly on intermediate viewports for cards, you'll need to check them again with Nick to define the correct behaviour